ansys/mapdl/core/mapdl_grpc.py

```diff
@@ -1884,3 +1884,155 @@ def _distributed(self) -> bool:
         if self.__distributed is None:
             self.__distributed = self.parameters.numcpu > 1
         return self.__distributed
+
+    @wraps(_MapdlCore.ndinqr)
+    def ndinqr(self, node, key, **kwargs):
+        """Wrap the ``ndinqr`` method to take advantage of the gRPC methods."""
+        pname = '__tmpvar__'
+        super().ndinqr(node, key, pname=pname, mute=True, **kwargs)
+        value = self.scalar_param(pname)
+        return value
+
+    @wraps(_MapdlCore.elmiqr)
+    def elmiqr(self, ielem, key, **kwargs):
+        """Wrap the ``elmiqr`` method to take advantage of the gRPC methods."""
+        pname = '__tmpvar__'
+        super().elmiqr(ielem, key, pname=pname, mute=True, **kwargs)
+        value = self.scalar_param(pname)
```

Comment:
This can be further simplified by defining `pname` as `TMPVAR` at the top of this module and use it as a "module constant."
```
TMPVAR = "__tmpvar__"
```
Then:
```suggestion
        super().elmiqr(ielem, key, pname=TMPVAR, mute=True, **kwargs)
        value = self.scalar_param(TMPVAR)
```

Reply:
Done.